modules/dists/PrivateDist.chpl

```diff
@@ -147,7 +147,7 @@ class PrivateDom: BaseRectangularDom {
 
 class PrivateArr: BaseRectangularArr {
   var dom: unmanaged PrivateDom(rank, idxType, stridable);
-  var data: eltType;
+  var data: eltType?;
```

Comment:
This does not seem right to me. Doesn't it mean that accessing a PrivateArr (via something dmapped Private) will give you back a nilable element even if you asked for a non-nilable element type?

I would expect code using PrivateDist with classes to use a nilable type when creating the Private array of classes.

Reply:
Good point. I adjusted.